go/libkb/log_send.go

```diff
@@ -500,3 +507,26 @@ func (l *LogSendContext) LogSend(statusJSON, feedback string, sendLogs bool, num
 
 	return l.post(statusJSON, feedback, kbfsLog, svcLog, desktopLog, updaterLog, startLog, installLog, systemLog, gitLog, watchdogLog, traceBundle, uid, installID)
 }
+
+// mergeExtendedStatus adds the extended status to the given status json blob.
+// If any errors occur the original status is returned unmodified.
+func (l *LogSendContext) mergeExtendedStatus(status string) string {
+	var statusObj map[string]interface{}
+	if err := json.Unmarshal([]byte(status), &statusObj); err != nil {
+		return status
+	}
+
+	extStatus, err := GetExtendedStatus(context.TODO(), l.G())
```

Comment:
As discussed in chat, I think we can do this simpler without the need of this library.

Reply:
done